pkg/openshift/rosa/accountroles.go

```diff
@@ -174,7 +184,10 @@ func (r *Provider) getAccountRoles(ctx context.Context, prefix, version string)
 	switch {
 	case accountRolesFound == 0:
 		return nil, nil
-	case accountRolesFound != 7:
+	case isFedRamp && accountRolesFound == fedRampRolesCount:
+		// Rosa blocks the creation of HCP roles for FedRamp clusters
+		return roles, nil
+	case accountRolesFound != commercialRolesCount:
 		return nil, fmt.Errorf("one or more prefixed %q account roles does not exist: %+v", prefix, roles)
 	default:
 		return roles, nil
```

Comment:
I don't understand this switch statement completely, we return the same thing in both the new case and the default case

Reply:
Ah gotcha